3rdparty/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ if (ECM_FOUND)
     include(ECMEnableSanitizers)
 endif()
 
-if(HEAPTRACK_BUILD_TRACK)
+if(HEAPTRACK_BUILD_INTERPRET)
```

Comment:
the code to add the optional BUILD_INTERPRET flag handling should be a separate patch